backend/resources.py

```diff
@@ -12,3 +12,4 @@ class ResourceName(Enum):
     PROCESSING_ASSETS_TABLE_NAME = f"{ENV}-processing-assets"
     STORAGE_BUCKET_NAME = f"{ENV}-linz-geospatial-data-lake"
     VALIDATION_RESULTS_TABLE_NAME = f"{ENV}-validation-results"
+    USERS_ROLE_NAME = f"{ENV}-data-lake-users"
```

Comment:
This should be above the previous line to be in order

Reply:
Thanks, fixed and rebased.